.github/workflows/create-release-pr.yml

```diff
@@ -13,6 +13,9 @@ on:
       release-version:
         description: 'A specific version to bump to. Mutually exclusive with "release-type".'
         required: false
+      version-synchronization-strategy:
+        description: 'Monorepos only. Whether and how to synchronize monorepo package versions. See Action readme for details.'
+        required: false
```

Comment:
would it make more sense for this to be `type: choice` with:
```
options:
        - all
        - transitive
        - internalOnly
        - none
 ```
 ?
 
 this also isn't fully what i'd expect. this is typically just `independent` and `fixed`, non?

Reply:
Ah, I didn't realize that input type was a thing. I'll definitely use that.

Regarding the available strategies, I discuss the available options in the description of this PR. There is a case to remove `internalOnly`, and I increasingly think that we should do so.

Regarding your suggested strategies, I'm guessing that `independent` (i.e., "any two versions may be different") would be equivalent to `none` and `fixed` (i.e., "all versions are always the same") to `all`. The addition of the `transitive` strategy was the motivating factor for this PR, and I don't think its meaning is captured by `independent` and `fixed`. This is also motivated in the PR description.